Realm/RLMSyncUtil_Private.hpp

```diff
@@ -35,6 +40,22 @@ std::shared_ptr<realm::SyncSession> sync_session_for_realm(RLMRealm *realm);
 
 CocoaSyncUserContext& context_for(const std::shared_ptr<realm::SyncUser>& user);
 
+#pragma mark - Access level conversion
+
+realm::AccessLevel accessLevelForObjcAccessLevel(RLMSyncAccessLevel level);
```

Comment:
This should probably be `accessLevelForObjCAccessLevel`.

Reply:
Good catch. Thank you.